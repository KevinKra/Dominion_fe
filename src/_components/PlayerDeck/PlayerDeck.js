import React from "react";
import "./PlayerDeck.scss";
import Card from "../Card/Card";

export default function PlayerDeck() {
  return (
    <section className='PlayerDeck'>
      <Card />
    </section>
  );
}
