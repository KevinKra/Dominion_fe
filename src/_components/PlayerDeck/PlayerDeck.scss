.PlayerDeck {
  border: 1px solid blue;
  position: absolute;
  z-index: 1;
  bottom: 15%;
  left: 2%;
  transition: 0.2s all ease;
}

@media screen and (max-width: 1500px) {
  .PlayerDeck {
    // width: 50%;
    transform: translate(115%, 15%);
  }
}
