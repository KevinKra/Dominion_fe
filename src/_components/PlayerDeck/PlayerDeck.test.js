import React from "react";
import { PlayerDeck } from "./PlayerDeck";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("<PlayerDeck />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<PlayerDeck />);
  });
  it("should match the snapshot", () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
