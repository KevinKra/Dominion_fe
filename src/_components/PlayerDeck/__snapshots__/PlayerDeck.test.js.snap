// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayerDeck /> should match the snapshot 1`] = `
<section
  className="PlayerDeck"
>
  <Card
    category="Treasure"
    cost={0}
    id={1113}
    image="Copper.jpg"
    key="1113"
    name="Copper"
  />
</section>
`;
